streaming/base/converters/dataframe_to_mds.py

```diff
@@ -289,11 +247,16 @@ def write_mds(iterator: Iterable):
     partitions = dataframe.mapInPandas(func=write_mds, schema=result_schema).collect()
 
     if merge_index:
-        do_merge_index(partitions, mds_path)
+        folder_urls = []
+        for row in partitions:
+            if ',' in row['mds_path']:
```

Comment:
this if-else block is also because of the tuple pickling?

Reply:
Yes, it is.